easy-ui-react/src/SectionCard/SectionCard.module.scss

```diff
@@ -0,0 +1,32 @@
+@use "../styles/common" as *;
+@use "../Tabs/mixins" as Tabs;
+
+.header {
+  display: flex;
+  flex-wrap: nowrap;
+  align-items: center;
+  justify-content: space-between;
+  min-width: 0;
+
+  // Ensure middle content is centered
+  > :nth-child(1),
+  > :nth-child(3) {
```

Comment:
**question**: is there a mockup that indicates a `HorizontalStack` wouldn't work for `SectionCard.Header`?

Reply:
`HorizontalStack` doesn't work well currently for true-centering the middle column. i'm going to look into a better solution for these cases